open-sphere-base/core/src/main/java/io/opensphere/core/NetworkConfigurationManager.java

```diff
@@ -71,15 +52,11 @@
     void restoreDefaults();
 
     /**
-     * Set the proxy configuration.
-     *
-     * @param host the host, or {@code ""} to disable the proxy
-     * @param port the port number
-     * @param useSystemProxies {@code true} if system proxies should be used
-     * @param configUrl the config URL
-     * @param hostPatterns the string comprising the host patterns
+     * @param configurationType
      */
-    void setProxyConfiguration(String host, int port, boolean useSystemProxies, String configUrl, String hostPatterns);
+    void setSelectedProxyType(ConfigurationType configurationType);

```

Comment:
Does this have comments github's diff is weird.

Reply:
It didn't, so, fixed.